docs/resources/samples/excel-cross-reference.md

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Cross reference and format an Excel file'
+description: 'Learn how to use Office Scripts and Power Automate to cross reference and format an Excel file.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Cross reference and format an Excel file
+
+This solution shows how two Excel files can be cross-referenced and formatted using Office Scripts and Power Automate.
+
+The project achieves the following:
+
+1. Extracts event data from <a href="events.xlsx">events.xlsx</a> using one Run script action.
+1. Passes that data to the second Excel file containing event transaction data and uses that data to do basic validation of data and formatting of missing or incorrect data using Office Scripts.
+1. Emails the result to a reviewer.
+
+For further details, see [Cross Reference and formatting two Excel files using Office Scripts](https://powerusers.microsoft.com/t5/Power-Automate-Cookbook/Cross-Reference-and-formatting-two-Excel-files-using-Office/td-p/728535).
+
+## Sample Excel files
+
+1. <a href="events.xlsx">events.xlsx</a>
+1. <a href="event-transactions.xlsx">event-transactions.xlsx</a>
+
+## Office Scripts
+
+1. [Get event data](#get-event-data)
+1. [Validate event transactions](#validate-event-transactions)
+
+### Get event data
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook): EventData[] {
+    let table = workbook.getWorksheet('Keys').getTables()[0];
+    let range = table.getRangeBetweenHeaderAndTotal();
+    let rows = range.getValues();
+    let records: EventData[] = [];
+    for (let row of rows) {
+        let [event, date, location, capacity] = row;
+        records.push({
+            event: event as string,
+            date: date as number, 
+            location: location as string,
+            capacity: capacity as number
+        })
+    }
+    console.log(JSON.stringify(records))
+    return records;
+}
+
+interface EventData {
+    event: string
+    date: number
+    location: string
+    capacity: number
+}
+```
+
+### Validate event transactions
+
+```TypeScript
```

Comment:
consider adding code comments

Reply:
Yeah, I had received this feedback before. We're tracking with this item: 4958338